server/middlewares/BookValidator.js

```diff
@@ -54,7 +54,9 @@ export default class BookValidator {
     return UserValidator.genericCheck('isbn')
       .trim()
       .isISBN()
-      .withMessage('isbn is not valid');
+      .withMessage('isbn is not valid')
+      .isInt({ min: 1000000000, max: 9999999999999 })
```

Comment:
This is not an ISBN, for more information check here https://en.wikipedia.org/wiki/International_Standard_Book_Number

Reply:
Speak to biola.

Reply:
kindly revert back so biola can create a bug fix for this or you can assist him